merlin/main.py

```diff
@@ -642,6 +708,27 @@ def setup_argparse():
         help="Check for active workers on an allocation.",
         formatter_class=RawTextHelpFormatter,
     )
+    monitor.add_argument(
+        "specification", type=str, help="Path to a Merlin YAML spec file"
+    )
+    monitor.add_argument(
+        "--steps",
+        nargs="+",
+        type=str,
+        dest="steps",
+        default=["all"],
+        help="The specific steps in the YAML file you want to query",
```

Comment:
Can we update this help string to be more specific to `monitor`?

Reply:
Yes, I will change this.